src/routes/safe/components/Transactions/helpers/TxModalWrapper/index.tsx

```diff
@@ -109,9 +110,10 @@ export const TxModalWrapper = ({
   const showCheckbox = !isSpendingLimitTx && canTxExecute && (!txThreshold || txThreshold > confirmationsLen)
   const nativeCurrency = getNativeCurrency()
   const { currentVersion: safeVersion, threshold } = useSelector(currentSafe) ?? {}
-  const { smartContractWallet } = useSelector(providerSelector)
   const isCreation = isMultisigCreation(confirmationsLen, txType)
-  const isOffChainSignature = checkIfOffChainSignatureIsPossible(doExecute, smartContractWallet, safeVersion)
+  const isSmart = useIsSmartContract(userAddress)
```

Comment:
I would be more explicit here with the name. Something like `isSmartContract` or `isSmartContractWallet`